src/components/CalendarMonth.jsx

```diff
@@ -132,10 +132,13 @@ export default class CalendarMonth extends React.Component {
 
     return (
       <div className={calendarMonthClasses} data-visible={isVisible}>
-        <table>
-          <caption className="CalendarMonth__caption js-CalendarMonth__caption">
-            <strong>{monthTitle}</strong>
-          </caption>
+        <div
+          id="CalendarMonth__caption"
+          className="CalendarMonth__caption js-CalendarMonth__caption"
+        >
+          <strong>{monthTitle}</strong>
+        </div>
+        <table role="presentation">
```

Comment:
Added the role of presentation here to stop the screen reader from reading all of the details of the columns and rows of the table [WAI-ARIA use of presentation role](https://www.w3.org/TR/2009/WD-wai-aria-practices-20090224/#kbd_layout_impact_mode_css2).